public/index.html

```diff
@@ -2,9 +2,11 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Mozfest 2015</title>
+  <title>MozFest 2015</title>
   <link rel="stylesheet" href="index.css"/>
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="Three days of building a truly global web together. Come with an idea, leave with a community.">
+  <meta property="og:description" content="Three days of building a truly global web together. Come with an idea, leave with a community.">
```

Comment:
i remember it will get overridden. but let me see if i can find the doc just to double check